test/test_distributions.py

```diff
@@ -1603,6 +1603,39 @@ def ref_log_prob(idx, x, log_prob):
 
         self._check_log_prob(Normal(loc, scale), ref_log_prob)
 
+    def test_normal_shape(self):
```

Comment:
nit (not blocking): I prefer the testing of this function to be in test_torch.py, because torch.normal is a function on torch.*, not a function on torch.distributions. But that's a pretty minor detail; don't worry about it if you don't want to